authors/apps/profiles/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import include, path
+from rest_framework import routers
+from rest_framework.urlpatterns import format_suffix_patterns
+
+# local imports
+from . import views
+
+router = routers.DefaultRouter()
+
+urlpatterns = [
```

Comment:
you ought to add the url path for confirmation email here as well :)

Reply:
Ok i will refactor